Marlin/src/pins/stm32f4/pins_LERDGE_K.h

```diff
@@ -197,7 +217,7 @@
 //
 #define SDSS                                PC11
 #define LED_PIN                             PA15  // Alive
-#define PS_ON_PIN                           -1
+#define PS_ON_PIN                           PA4
```

Comment:
I don't see a dedicated pin on the board for turning the power supply on and off. This pin coincides with the `POWER_LOSS_PIN` and I the same pin cannot be used for both `M80`/`M81` and power-loss detection at the same time.

Reply:
> How can I check the `POWER_LOSS` function?

The `POWER_LOSS` pin will be pulled `HIGH` or `LOW` depending on what is connected to it, and the pin will go `LOW` or `HIGH` in response to the connected device. If the board has a large capacitor onboard, maybe there is a circuit that can detect loss of current to the board, with enough current left in the capacitor to allow a file write to the SD card.

You can test whether this pin is detecting a power loss to the board by monitoring it in `loop()` and printing a message to serial whenever its state changes. If the state changes when you remove power from the board, and there is enough time for the serial output, then we know it works.